src/styles.css

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~bootstrap/dist/css/bootstrap.css";
```

Comment:
This works but the ideal place for this would be the `styles` array inside the `angular.json` file